package.json

```diff
@@ -626,6 +627,60 @@
                                         ]
                                     }
                                 }
+                            },
+                            "python": {
+                                "description": "Options for building Python applications in Docker.",
+                                "properties": {
+                                    "host": {
+                                        "type": "string",
+                                        "description": "The host for remote debugging.",
+                                        "default": "localhost"
+                                    },
+                                    "port": {
+                                        "type": "number",
+                                        "description": "The port for remote debugging.",
+                                        "default": 5678
+                                    },
+                                    "pathMappings": {
+                                        "type": "array",
+                                        "items": {
+                                            "type": "object",
+                                            "properties": {
+                                                "localRoot": {
+                                                    "type": "string",
+                                                    "description": "The host path."
+                                                },
+                                                "remoteRoot": {
+                                                    "type": "string",
+                                                    "description": "The container path."
+                                                }
+                                            }
+                                        }
+                                    },
+                                    "justMyCode": {
+                                        "type": "boolean",
+                                        "description": "Debug only user-written code."
+                                    },
+                                    "django": {
```

Comment:
Should `django` / `jinja` be an enum rather than two booleans? Or can both be true at the same time?

Reply:
Technically they both can be true at the same time, no idea how will that behave though. I followed the same syntax in the Python extension just to ensure parity.